### PR TITLE
Fix a variety of minor type errors so tsc runs cleanly.

### PR DESCRIPTION
**Pull request by reem**
Fixes a few minor type errors that pop up when running `yarn run tsc --noEmit` locally. Also changes the type of `getGovernance` so it's consumable within aave-proposals-v3 without causing type errors.

### CODE REVIEW

File: src/reports/fetch-IR-strategy.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const streamPipeline = promisify(pipeline);
  * @param {*} rate
  * @param fileName the fileName to store the ir to
  */
-export async function fetchRateStrategyImage(rate) {
+export async function fetchRateStrategyImage(rate: any) {
```
Comment (sakulstra) [inline src/reports/fetch-IR-strategy.ts:16]: I think we would have a type for this as well here: https://github.com/bgd-labs/aave-cli/blob/main/src/reports/snapshot-types.ts#L66 would you mind adding?

File: tsconfig.json [hunk 2]
```diff
@@ -2,8 +2,9 @@
   "compilerOptions": {
     "strict": true,
     "target": "ESNext",
-    "moduleResolution": "Node",
+    "moduleResolution": "node",
```
Comment (sakulstra) [inline tsconfig.json:5]: just curious, does this matter, no right?

File: src/utils/tenderlyClient.ts [hunk 3]
```diff
@@ -539,7 +539,7 @@ class Tenderly {
 }
 
 export const tenderly = new Tenderly(
-  process.env.TENDERLY_ACCESS_TOKEN,
-  process.env.TENDERLY_ACCOUNT,
-  process.env.TENDERLY_PROJECT_SLUG
+  process.env.TENDERLY_ACCESS_TOKEN!,
```
Comment (sakulstra) [inline src/utils/tenderlyClient.ts:542]: weird, did you get a ts error before? As i think you should not: https://github.com/bgd-labs/aave-cli/blob/main/types.d.ts#L3

perhaps you do as your change on tsconfig only includes src? I'm not so familiar with all these config options.

File: tsconfig.json (hunk 2 above)
Comment (reem) [reply tsconfig.json:5]: I didn't actually see a difference here no, I just did it for consistency as I was trying to figure out some other config changes

File: src/utils/tenderlyClient.ts (hunk 3 above)
Comment (reem) [reply src/utils/tenderlyClient.ts:542]: yup this was due to the tsconfig change, moving the types file into src also fixes

**Comment by reem** [comment]
Updated for review comments

**Review by sakulstra — APPROVED** [review]

**Pull request merged**